notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<p>For such a small list (3 elements), I would prefer to have them shown explicitly like ['xPosition', 'yPostion', 'zPosition'] and so on.</p><p> </p><p>This is just a personal preference, so no need to consider if you don't want to.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>I received an opposite comment in the review of another notebook ;-)  But I agree with you and I think that writing the list elements explicitly is clearer --&gt; done </p>